.idea/almanac.iml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
you'll want to merge master in, which has your change to add these to `.gitignore`